src/os/bluestore/BitMapAllocator.cc

```diff
@@ -221,8 +220,8 @@ void BitMapAllocator::init_rm_free(uint64_t offset, uint64_t length)
            << " length 0x" << length << std::dec
            << dendl;
 
-  assert(!(offset % m_block_size));
```

Comment:
@chhabaramesh 

The **IS_P2ALIGNED(v, a)** macro requires the given value **v** to be type **a** aligned, and type **a** must be of power of 2 aligned( which means **IS_P2ALIGNED(v, a)** does not require **v** to be power of 2 of **a** aligned). 

There are examples showing how this macro works:
```
 /*
 * Macro for checking power of 2 address alignment.
 * eg, IS_P2ALIGNED(1200, 1024) == false
 * eg, IS_P2ALIGNED(1024, 1024) == true
 * eg, IS_P2ALIGNED(0, 1024) == true
 * eg, IS_P2ALIGNED(0x1234, 0x100) == false
 * eg, IS_P2ALIGNED(0x5600, 0x100) == true
 */

```

The following test case is positive also:
``` c++
  if (IS_P2ALIGNED(3 * 1024, 1024))
    cout << "aligned!" << endl;
```
result:
```
aligned!